### DISCUSSION TITLE
Mod vweps dont show in mvd demo playback

### DISCUSSION THREAD

**Discussion by jkr4m3r** [ezQuake/ezquake-source#667] [Ideas]
Specifically Painkeep, I was told long ago they would never work even though the models were named properly, etc.

Is this still the case? Hopeful since I can see the vanilla QW vweps in mvd demos.

**Comment by kendravngrps006-gif** [comment]
Momentan sind Painkeep-Modelle noch schwierig – nur korrekt benannt zu sein, reicht nicht immer, damit sie reibungslos funktionieren. Einige Nutzer berichten von Teilerfolgen, besonders wenn Vanilla-QW-Waffen in bassbet-de.com.de MVD-Demos https://bassbet-de.com.de/ erscheinen, aber volle Kompatibilität erfordert oft Anpassungen oder eigene Skripte. Ein Test lohnt sich, doch eine garantierte Plug-and-Play-Erfahrung ist derzeit noch nicht gegeben.

**Reply by jkr4m3r** [reply]
New account 18 minutes ago, this comment posted 15 minutes ago? Sounding like an ai answer?

Suspicious.

**Reply by dsvensson** [reply]
Nopes, works fine. You just need the vwplayer.mdl etc and it's patched when the demo loads. Don't recall if it's limited to mvd or it works in the qwd path as well. Think the nQuake package contains all that's needed.

**Reply by jkr4m3r** [reply]
Has never worked for Painkeep mvd demos since 2010 that I know of. I asked one of the previous ezquake devs years ago about this, and he said it was a hack on their end that would only work for vanilla quake weapons, not Painkeep weapons. Johnny_cz was his name IIRC.

[edit] vweps have always worked fine in qwd demos, btw.

[edit2] To be clear, I'm referring to seeing the vweps from 3rd person, such as a spectator might see.

**Comment by JesseReed123** [comment]
Pour Painkeep, il est vrai qu’il y a longtemps, on disait que certains modèles ne fonctionnaient pas, même si les fichiers étaient correctement nommés. Cependant, avec les mises à jour récentes et les démonstrations MVD, on peut voir que les armes QW originales apparaissent correctement, ce qui laisse penser que certaines limitations ont été levées. Il reste recommandé [Roobet ](https://roobet.live/)de tester directement dans le mod pour vérifier la compatibilité et de consulter les forums de la communauté pour des astuces sur l’activation de ces modèles.

**Reply by jkr4m3r** [reply]
This feels like an AI response?

**Comment by RowanCross512** [comment]
As far as I know, that limitation hasn’t really changed. Painkeep still doesn’t play nicely with MVD demos in terms of custom vweps, even if the models are named correctly. What you’re seeing with vanilla QuakeWorld [minimum deposit casino australia](https://payiddeposit.com/minimum/) weapons working is expected, since those follow standard conventions the demo system understands. Painkeep adds its own quirks and assets, which don’t always get recorded or replayed properly in MVD. Unless there’s been a recent community workaround or patch, it’s likely still a no-go for full support.

**Reply by jkr4m3r** [reply]
Yeesh!
Joined today, and responded to this old thread? I hope AI doesn't ruin everything on the internet.

**Reply by Calinou** [reply]
Unfortunately, the entire Discussions tab for this repository is getting overrun with spam. It was largely inactive until a spam wave started appearing near the end of last year.
